_includes/rss_logo.html

```diff
@@ -1,13 +1,13 @@
-<svg width="30" height="30" viewBox="0 0 26 26" fill="none" xmlns="http://www.w3.org/2000/svg">
-  <g clip-path="url(#clip0)">
-  <path d="M3.60112 21.9555C5.56084 21.9555 7.14951 20.3669 7.14951 18.4072C7.14951 16.4474 5.56084 14.8588 3.60112 14.8588C1.6414 14.8588 0.0527344 16.4474 0.0527344 18.4072C0.0527344 20.3669 1.6414 21.9555 3.60112 21.9555Z" fill="white"/>
-  <path d="M11.8871 20.217C11.8871 21.1928 12.6855 21.9911 13.6613 21.9911C14.6371 21.9911 15.4355 21.1928 15.4355 20.217C15.4355 12.6944 9.31452 6.55566 1.77419 6.55566C0.798387 6.55566 0 7.35405 0 8.32986C0 9.30566 0.798387 10.1041 1.77419 10.1041C7.3629 10.1041 11.8871 14.646 11.8871 20.217Z" fill="white"/>
-  <path d="M20.2256 21.9912C21.2015 21.9912 21.9998 21.1928 21.9998 20.217C21.9998 9.07504 12.9337 0.00891113 1.79177 0.00891113C0.815965 0.00891113 0.0175781 0.807298 0.0175781 1.7831C0.0175781 2.75891 0.815965 3.5573 1.79177 3.5573C10.9821 3.5573 18.4515 11.0267 18.4515 20.217C18.4515 21.1928 19.2321 21.9912 20.2256 21.9912Z" fill="white"/>
-  </g>
-  <defs>
-  <clipPath id="clip0">
-  <rect width="26" height="26" fill="white"/>
-  </clipPath>
-  </defs>
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="30" height="30" fill="none" viewBox="0 0 26 26" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I don't think width and height attributes are needed, you should be able to remove them and keep only viewBox